onebusaway-android/src/main/java/org/onebusaway/android/map/RouteMapController.java

```diff
@@ -211,6 +212,33 @@ public void onSaveInstanceState(Bundle outState) {
         outState.putString(MapParams.ROUTE_ID, mRouteId);
         outState.putBoolean(MapParams.ZOOM_TO_ROUTE, mZoomToRoute);
         outState.putBoolean(MapParams.ZOOM_INCLUDE_CLOSEST_VEHICLE, mZoomIncludeClosestVehicle);
+
+        Location centerLocation = mFragment.getMapView().getMapCenterAsLocation();
+        outState.putDouble(MapParams.CENTER_LAT, centerLocation.getLatitude());
+        outState.putDouble(MapParams.CENTER_LON, centerLocation.getLongitude());
+        outState.putFloat(MapParams.ZOOM, mFragment.getMapView().getZoomLevelAsFloat());
+        Log.v("save zoomlevel", mFragment.getMapView().getZoomLevelAsFloat() + "");
```

Comment:
Let's delete this Log statement - seems only useful in debugging to make sure this implementation works correctly.